platform/linux-generic/odp_init.c

```diff
@@ -45,8 +45,8 @@ enum init_stage {
 	ALL_INIT      /* All init stages completed */
 };
 
-struct odp_global_data_ro_t odp_global_ro;
-struct odp_global_data_rw_t *odp_global_rw;
+odp_global_data_ro_t odp_global_ro;
+odp_global_data_rw_t *odp_global_rw;
```

Comment:
These global variables should be prefixed with underscore.

Reply:
It's better to do variable renaming after this as touches many files not related to this patch set.